packages/qdrant-loader-core/src/qdrant_loader_core/graph/__init__.py

```
@@ -0,0 +1,43 @@
+
+        if get_settings is not None:
+            try:
+                settings = get_settings()
+                graph_cfg = getattr(settings.global_config, "graph", None)
+                if graph_cfg is not None:
+                    host = host or graph_cfg.connection.host
+                    port = port or str(graph_cfg.connection.port)
+                    graph_name = graph_name or graph_cfg.graph_name
+            except Exception:
+                # Settings may not be initialized in some runtime contexts.
+                pass
```

Ruff (0.15.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)